### PR TITLE
Change circuit to return Unhealthy

### PR DESCRIPTION
**Pull request by Raynos**
This changes the CircuitBreaker to return an Unhealth
error frame instead of a Declined error frame.

This will:

 - Stop retries on circuit broken; to avoid retry storm
 - Make graphs a bit nicer because Unhealthy is only
	used by circuit breaker.

r: @rf @kriskowal @jcorbin

### CODE REVIEW

**Comment by jcorbin** [comment]
Please make it configurable in a way similar to how circuit shorting is configured: on `cn~sn~arg1` tuples, with simple wildcarding.  In same push, it will serve to shift the boundary away from `func (Circuit) shouldRequest() bool` to `func (Circuit) getRequestError() (code, message string)`.

**Comment by rf** [comment]
lgtm

File: circuits.js [hunk 1]
```diff
@@ -98,7 +97,8 @@ function Circuits(options) {
     this.statsd = options.statsd;
     this.circuitsByServiceName = {};
     this.config = options.config || {};
-    this.shorts = null;
+    this.shorts = options.shorts;
```
Comment (jcorbin) [inline circuits.js]: || null

**Comment by jcorbin** [comment]
Recording our offline convo: please make it actually flippable per circuit, not just for all circuits.

**Comment by jcorbin** [comment]
This looks :ok: to me now; ideally we'd split out:
- simplify the shorts config (originally I had it setup to thread thru, but then backed out to referencing off root only)
- change the circuit api to return error frame / reject with it
- the rest?

But I can live with this as is if you're pressed for time.

**Pull request merged**